internal/dataplane/parser/translators/ingress_test.go

```diff
@@ -1304,82 +1300,6 @@ func TestTranslateIngress(t *testing.T) {
 	}
 }
 
-func TestPathsFromIngressPaths(t *testing.T) {
-	for _, tt := range []struct {
-		name string
-		in   netv1.HTTPIngressPath
-		out  []*string
-	}{
-		{
-			name: "path type prefix will expand the match to a trailing slash if not provided",
-			in: netv1.HTTPIngressPath{
-				Path:     "/v1/api/packages",
-				PathType: &pathTypePrefix,
-			},
-			out: kong.StringSlice(
-				"/v1/api/packages/",
-				"/v1/api/packages$",
-			),
-		},
-		{
-			name: "path type prefix will expand the match with a literal match if a slash is provided",
-			in: netv1.HTTPIngressPath{
-				Path:     "/v1/api/packages/",
-				PathType: &pathTypePrefix,
-			},
-			out: kong.StringSlice(
-				"/v1/api/packages/",
-				"/v1/api/packages$",
-			),
-		},
-		{
-			name: "path type prefix will provide a default when no path is provided",
-			in: netv1.HTTPIngressPath{
-				Path:     "",
-				PathType: &pathTypePrefix,
-			},
-			out: kong.StringSlice("/"),
-		},
-		{
-			name: "path type exact will cause an exact matching path on a regular path",
-			in: netv1.HTTPIngressPath{
-				Path:     "/v1/api/packages",
-				PathType: &pathTypeExact,
-			},
-			out: kong.StringSlice("/v1/api/packages$"),
-		},
-		{
-			name: "path type exact will cause an exact matching path on a regular path with a / suffix",
-			in: netv1.HTTPIngressPath{
-				Path:     "/v1/api/packages/",
-				PathType: &pathTypeExact,
-			},
-			out: kong.StringSlice("/v1/api/packages/$"),
-		},
-		{
-			name: "path type exact will supply a default if no path is provided",
-			in: netv1.HTTPIngressPath{
-				Path:     "",
-				PathType: &pathTypeExact,
-			},
-			out: kong.StringSlice("/$"),
-		},
-		{
-			name: "path type implementation-specific will leave the path alone",
-			in: netv1.HTTPIngressPath{
-				Path:     "/asdfasd9jhf09432$",
-				PathType: &pathTypeImplementationSpecific,
-			},
-			out: kong.StringSlice("/asdfasd9jhf09432$"),
-		},
-	} {
-		t.Run(tt.name, func(t *testing.T) {
-			// TODO split test cases to handle regex
```

Comment:
Could we instead of dropping this test completely, convert it to call `PathsFromIngressPaths(tt.in)` and align the expected paths?

Reply:
It duplicates `TestPathsFromIngressPathsRegexPrefix`, maybe I should rename `TestPathsFromIngressPathsRegexPrefix` -> `TestPathsFromIngressPaths`. WDYT?

Reply:
Cool 👍